include/swift/AST/DiagnosticsSema.def

```diff
@@ -1367,6 +1367,10 @@ ERROR(pattern_binds_no_variables,none,
       "variables",
       (unsigned))
 
+ERROR(stored_property_no_uninhabited_type,none,
+      "stored property %0 cannot have uninhabited type %1",
```

Comment:
I don't love this diagnostic because I don't think "uninhabited" is a great user-facing term, but I can't think of anything really better.

Reply:
Yea, I was looking for ways to reword this, but I couldn't come up with anything.

Reply:
The compiler seems to define uninhabited types as either enums with no cases, or tuples containing uninhabited types. Could we just call it a "caseless enum" or "enum with no cases"? I suppose that might require a separate phrasing for tuples:

```
ERROR(stored_property_no_uninhabited_enum,none,
      "stored property %0 cannot have caseless enum %1", ...)
 ERROR(stored_property_no_uninhabited_tuple,none,
      "stored property %0 cannot have tuple type %1 containing caseless enum", ...)
 ```

Reply:
How does this sound?
```
// Non-structurally
"%select{%select{variable|constant}0|property}1 %2 cannot have "
"enum type %3 with no cases"
// structurally
"%select{%select{variable|constant}0|property}1 %2 cannot have "
"tuple type %3 containing enum with no cases"
```